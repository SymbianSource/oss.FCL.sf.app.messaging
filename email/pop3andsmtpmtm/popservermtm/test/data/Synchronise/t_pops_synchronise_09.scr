+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted 
+OK 5 6415
+OK 5 messages
1 1314
2 1275
3 1234
4 1317
5 1275
.
+OK 5 messages
1 t_pops_synchronise:message002
2 t_pops_synchronise:message005
3 t_pops_synchronise:message006
4 t_pops_synchronise:message007
5 t_pops_synchronise:message009
.
-ERR Unknown Command "CAPA"
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message009; Wed, 25 Jul 2001 12:01:06 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1118058; 25 Jul 2001 12:01 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6a0aba7@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 13:16:17 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD792CP>; Wed, 25 Jul 2001 13:05:09 +0100
Message-ID: <t_pops_synchronise_message009@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 9
Date: Wed, 25 Jul 2001 12:58:33 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Nine Nine Nine Nine Nine Nine Nine Nine Nine Nine 
.
** Read cancelled
+OK 
