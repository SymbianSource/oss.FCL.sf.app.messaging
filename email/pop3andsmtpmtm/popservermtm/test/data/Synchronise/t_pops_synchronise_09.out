STAT
LIST
UIDL
CAPA
TOP 5 1
QUIT
